recipes/folly/all/conanfile.py

```diff
@@ -76,7 +77,7 @@ def requirements(self):
         self.requires("glog/0.4.0")
         self.requires("libevent/2.1.12")
         self.requires("lz4/1.9.3")
-        self.requires("openssl/1.1.1i")
+        self.requires("openssl/1.1.1k")
         self.requires("snappy/1.1.8")
         self.requires("zlib/1.2.11")
         self.requires("zstd/1.4.8")
```

Comment:
1.5.0 is out AFAIK... might be a safe update

Reply:
folly itself uses 1.4.5: https://github.com/facebook/folly/blob/master/build/fbcode_builder/manifests/zstd#L12

Not sure it is a good idea to run ahead of ourselves.